react/propTypes.js

```diff
@@ -0,0 +1,10 @@
+import PropTypes from 'prop-types'
+
+export const appShape = PropTypes.shape({
+  slug: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  namePrefix: PropTypes.string,
+  comingSoon: PropTypes.bool,
+  href: PropTypes.string,
+  links: PropTypes.shape({ icon: PropTypes.string.isRequired })
+})
```

Comment:
The idea is to write all our UI proptypes here?

Reply:
Not necessarily, mostly those which are shared. AppLinker is a refactor of a component from the bar (AppItem), that had this propTypes, this is why I added it here.